### PR TITLE
Create proc_creation_win_deviceenroller_evasion.yml

### PR DESCRIPTION
**Pull request by danielgottt**

### CODE REVIEW

File: rules/windows/process_creation/proc_creation_win_deviceenroller_evasion.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+logsource:
+  category: process_creation
+  product: windows
+detection:
+    selection:
+        ParentImage|endswith:
+            - '\cmd.exe'
+            - '\powershell.exe'
+        Image|endswith: '\deviceenroller.exe'
+        CommandLine|re: '(/PhoneDeepLink|/phonedeeplink)'
```
Comment (frack113) [inline rules/windows/process_creation/proc_creation_win_deviceenroller_evasion.yml:22]: sigma is case insensitive no need a regex here
`CommandLine|contains: '/PhoneDeepLink'`

File: rules/windows/process_creation/proc_creation_win_deviceenroller_evasion.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+date: 2022/08/29
+tags:
+  - attack.defense_evasion
+  - attack.t1574.002
+logsource:
+  category: process_creation
+  product: windows
+detection:
+    selection:
+        ParentImage|endswith:
```
Comment (frack113) [inline rules/windows/process_creation/proc_creation_win_deviceenroller_evasion.yml:18]: I think that ParentImage should not be usefull.

File: rules/windows/process_creation/proc_creation_win_deviceenroller_evasion.yml [hunk 3]
```diff
@@ -0,0 +1,26 @@
+title: DLL Sideloading via DeviceEnroller.exe
+id: e173ad47-4388-4012-ae62-bd13f71c18a8
+description: Recent research discovered that DeviceEnroller.exe will load the DLL ShellChromeAPI.dll which does not exist
```
Comment (frack113) [inline rules/windows/process_creation/proc_creation_win_deviceenroller_evasion.yml:3]: It is more
 "
Detect use of the parameter PhoneDeepLink to SideLoading....
Recent research....
"

File: rules/windows/process_creation/proc_creation_win_deviceenroller_evasion.yml (hunk 2 above)
Comment (danielgottt) [reply rules/windows/process_creation/proc_creation_win_deviceenroller_evasion.yml:18]: Sounds good! Will make change now

**Pull request merged**